build.gradle

```diff
@@ -6,7 +6,7 @@ ext {
 
   dep = [
       androidPlugin: 'com.android.tools.build:gradle:2.3.0-beta3',
-      apolloPlugin: "com.apollographql.android:gradle-plugin:0.1.0",
+      apolloPlugin: "com.apollographql.android:gradle-plugin:0.2.0-SNAPSHOT",
```

Comment:
I'm not a fan of having development outside of master, we can keep master always pointing to the SNAPSHOTS I think. This shouldn't be pointing to 0.1.0 in the first place